src/client/registry.ts

```diff
@@ -55,17 +55,17 @@ export function assertIsRegistryProviderInfo(
   }
 }
 
-export async function fetchMapAST(url: string): Promise<MapDocumentNode> {
-  const fetchInstance = new CrossFetch();
-  const http = new HttpClient(fetchInstance);
-  registryDebug('Fetching Map AST from registry');
-  const { body } = await http.request(url, {
-    method: 'GET',
-    accept: 'application/json',
-  });
+// export async function fetchMapAST(url: string): Promise<MapDocumentNode> {
```

Comment:
Do we want to keep it? 🤷

Reply:
I don't think so? Ideally I'd like to use service client or something later.

Plus, this wasn't being used (maps are always retrieved using bind, even when Provider is local).

Although maybe that should change?

Reply:
This will probably change with LUVs PR, right? U would just removed commented function 🤷